src/Akeneo/Platform/Job/front/process-tracker/src/feature/models/JobExecutionFilter.ts

```diff
@@ -13,6 +13,7 @@ type JobExecutionFilter = {
   sort: JobExecutionFilterSort;
   type: string[];
   status: JobStatus[];
+  users: string[];
```

Comment:
```suggestion
  user: string[];
```
For me the logic was to name it as filter, `type` filter, `status` filter and `user` filter, what do you think?

Reply:
I will add a `_filter` suffix as suggested by @StevenVAIDIE in another comment